### ISSUE TITLE
[BUG] Stuck at "Processing" after "Click Here To Setup" for 14 hours and counting

### ISSUE THREAD

**Issue by DoughyInTheMiddle** [lkwq007/stablediffusion-infinity#122]
After fighting with the install (had to uninstall Visual Studio and manually install fpie), I ran app.py manually in the prompt window. Taichi started, PIE initialized, and got the URL (curiously, the same URL as I'd have for regular SD, but whatever).

Opened the URL, bopped over to HugginFace to create a key, entered it in the box, and hit the "Click Here to Setup" button. Everything started spinning "Processing", and after waiting an hour, I realized, yeah, my potato PC/nv1060 be strugglin', so I went to bed. 

That was 12:30am last night. It's almost 1PM now. Sure, it says "This may take awhile" but this seems......excessive even for my machine. 

For perspective: When I process images in AUTOMATIC, most renders come out in about a minute. Not at all great, but survivable. So, if it's the GPU behind the "setup", maybe that explains it. Still, a full 12 hours?

**Desktop (please complete the following information):**
 - Windows 10
 - Brave
 - NVIDA GTX 1060 3GB (stop laughing!, lol)
 
![2022-10-27 12_59_05-StableDiffusion-Infinity - Brave](https://user-images.githubusercontent.com/44233670/198352977-3ffebf18-4074-4a43-81ff-01626ba48403.png)

**Comment by DoughyInTheMiddle** [comment]
Wiped the install just now because I had some errors. Followed a different install guide ([AItrepeneur](https://www.youtube.com/watch?v=ZPEwPEBCY1c)) and saw that I had made an error in conserving C: drive space by installing Anaconda on a drive other that C:. Uninstalled Anaconda, wiped the install, and did everything by hand again.

Unlike before, at least this time there's actual activity in the prompt window. Absolutely nothing was happening when I did it last night. 

Promising?

![image](https://user-images.githubusercontent.com/44233670/198399387-0b99818a-c56d-41a4-93c0-1afe8dd4d131.png)
